Competition2017/src/competition/subsystems/collector/CollectorSubsystem.java

```diff
@@ -0,0 +1,47 @@
+package competition.subsystems.collector;
+
+import org.apache.log4j.Logger;
+
+import com.ctre.CANTalon;
+import com.google.inject.Inject;
+import com.google.inject.Singleton;
+
+import xbot.common.command.BaseSubsystem;
+import xbot.common.controls.actuators.XCANTalon;
+import xbot.common.injection.wpi_factories.WPIFactory;
+import xbot.common.properties.DoubleProperty;
+import xbot.common.properties.XPropertyManager;
+@Singleton
+public class CollectorSubsystem extends BaseSubsystem {
+    private static Logger log = Logger.getLogger(CollectorSubsystem.class);
+    
+    private final DoubleProperty intakeSpeedProperty;
+    private final DoubleProperty ejectSpeedProperty;
+    
+    private final XCANTalon collectorMotor;
+    
+    @Inject
+    public CollectorSubsystem(WPIFactory factory, XPropertyManager propManager){
+        log.info("Creating Collector Subsystem");
+        collectorMotor = factory.getCANTalonSpeedController(5);
+        
+        collectorMotor.setBrakeEnableDuringNeutral(false);
+        
+        collectorMotor.setProfile(1);
+        collectorMotor.setControlMode(CANTalon.TalonControlMode.PercentVbus);
+        
+        ejectSpeedProperty = propManager.createPersistentProperty("Collector Eject Speed:", -0.5);
```

Comment:
Remove colons, and later words can be lowercased. Only capitalizing the first letter makes it easier to spot the start of a property.